tests/test_secret_order_status_cn.py

```
@@ -209,14 +170,14 @@ def test_recovered_grouped_normalizes_legacy_list():
     assert [o["id"] for o in out["待核议"]] == [2]
     for entry in out["在办"] + out["待核议"]:
-        assert "status" not in entry  # 旧档英文 status 在归一时剥掉
+        assert "status" not in entry
 
     assert _recovered_grouped(None) == {}
     assert _recovered_grouped("junk") == {}
 
 
 def test_resolve_context_roundtrips_grouped_secret_orders_as_dict(game):
-    """save→get resolve_context 保持分组 dict 形状（非空与空分组都不退成 list）。"""
+    """save→get resolve_context 保持分组 dict 形状（恢复端，非公共 LLM 隔离）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)